crates/cli-support/src/descriptor.rs

```diff
@@ -154,7 +154,27 @@ impl Descriptor {
         }
     }
 
-    pub fn get_64bit(&self) -> Option<bool> {
+    pub fn is_primitive(&self) -> bool {
```

Comment:
Could this perhaps be named slightly differently to something like `is_wasm_native_type` or something like that?